### PR TITLE
adding myself

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: Data/participants.json [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
+     "participants": [
+        {
+      "name": "Amit meena",
+      "email": "meenalko24@gmail.com",
+      "profilePicture": "https://avatars0.githubusercontent.com/u/56061943?s=400&u=afd34e94b98bf2e3755835dde859ca7acf59a87c&v=4",
+      "facebookLink": "https://www.facebook.com/amit.rock.79025",
+      "githubLink": "https://github.com/amitmeena24"
+        }
+  	  ]
+	}
```
Comment (dojutsu-user) [inline Data/participants.json]: Please don't edit the already ones.

File: update myself [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+  "participants": [
+    {
+      "name": "Amit meena",
+      "email": "meenalko24@gmail.com",
+      "profilePicture": "https://avatars0.githubusercontent.com/u/56061943?s=400&u=afd34e94b98bf2e3755835dde859ca7acf59a87c&v=4",
+      "facebookLink": "https://www.facebook.com/amit.rock.79025",
+      "githubLink": "https://github.com/amitmeena24"
```
Comment (dojutsu-user) [inline update myself:8]: This is not correct. Please see already merged PRs for reference.
For Example: #54

**Review by dojutsu-user — CHANGES_REQUESTED** [review]

**Comment by dojutsu-user** [comment]
Also, please resolve the merge conflicts.

**Comment by ghost** [comment]
sir ho hi ni rha

**Comment by ghost** [comment]
sir yeh lijiye data 

{
     "participants": [
        {
      "name": "Amit meena",
      "email": "meenalko24@gmail.com",
      "profilePicture": "https://avatars0.githubusercontent.com/u/56061943?s=400&u=afd34e94b98bf2e3755835dde859ca7acf59a87c&v=4",
      "facebookLink": "https://www.facebook.com/amit.rock.79025",
      "githubLink": "https://github.com/amitmeena24"
        }
  	  ]
	}

**Comment by dojutsu-user** [comment]
@amitmeena24 
You can close this PR.
Then make a separate branch and do the changes as done in already merged PRs.
Then make a PR.

> sir ho hi ni rha

Don't worry, you will learn a lot from this. :tada:

**Comment by ghost** [comment]
Ok sir

**Comment by rishabhshuklax** [comment]
Create a new PR from a new branch, I am closing this one. And don't worry you're doing great if you have any doubts you can personally contact any of us.

**Pull request closed without merging**